lib/remap-lang/src/function/split.rs

```diff
@@ -0,0 +1,90 @@
+use crate::{
+    expression::Error, Argument, ArgumentList, Expr, Expression, Function, Object, Parameter,
+    Result, State, Value,
+};
+use std::convert::{TryFrom, TryInto};
+
+#[derive(Debug)]
+pub(crate) struct Split;
```

Comment:
This function is more of a proof-of-concept.

For now, the idea is to leave all functions in the main Vector crate (they pull in quite a bit of external dependencies, and also use types defined in Vector itself).

In a follow-up PR this function will be moved back into Vector.